pandas/io/formats/style_render.py

```diff
@@ -57,17 +57,23 @@ class CSSDict(TypedDict):
 Subset = Union[slice, Sequence, Index]
 
 
+def _gl01_adjust(obj: Any) -> Any:
```

Comment:
umm this seems kind of a hack, can we just adjust the templates?

Reply:
These are the docstrings of `jinja2.PackageLoader` and `jinja2.Environment` I believe, and their docstrings don't conform with `GL01` cc @attack68

Reply:
thats right these are inherited docstring objects.

is there a way we can prevent the doc string validation is such cases? might be better than overwriting it?

Reply:
Thanks. Looks like the validation script doesn't have any capabilities to ignore certain objects or methods; it looks to check everything in the public API.

Reply:
ok merging

but maybe would be good to enable some ignoring to prevent having to do this